src/opera/compare/template_comparer.py

```diff
@@ -0,0 +1,137 @@
+from .diff import Diff
+from .comparisons import Comparison, ListComparison, MapComparison
+from opera.error import DataError
+
+from os import path
+import filecmp
```

Comment:
This imports section can be changed to:

```suggestion
import filecmp

from os import path

from .diff import Diff
from .comparisons import Comparison, ListComparison, MapComparison
from opera.error import DataError
```

to keep consistency with opera commands and files that split up the imports into three sections.